### ISSUE TITLE
Is there a plan to add  pairwise comparison but one is the result generated by ChatGPT itself?

### ISSUE THREAD

**Issue by Green-li** [MilkWhite/LLMs_for_Reference_Free_Text_Quality_Evaluation#1]
As the title, will you add the experiments about pairwise comparison(one result is from ChatGPT).

是否会考虑增加每条样本的结果都和ChatGPT的结果进行对比。
让ChatGPT首先给自己的输出打分，再给其他模型生成的结果打分。
这种方法相比于直接给出分数哪个更有效呢？

**Comment by MilkWhite** [comment]
Yeah, that's a good idea. We leave it for future work. It's hard to tell which one is better without experiments, and you can give it a try if interested.
